lib/widgets-customize.php

```diff
@@ -106,7 +106,34 @@ function gutenberg_widgets_customize_add_unstable_instance( $args, $id ) {
 	return $args;
 }
 
+/**
+ * Initialize the Gutenberg customize widgets page.
+ */
+function gutenberg_customize_widgets_init() {
+	$settings = array_merge(
+		gutenberg_get_default_block_editor_settings(),
+		gutenberg_get_legacy_widget_settings()
+	);
+
+	// This purposefully does not rely on apply_filters( 'block_editor_settings', $settings, null );
```

Comment:
Newly merged `gutenberg_get_block_editor_settings` is going to be a great fit here because it deprecates and more importantly simply ignores `block_editor_settings` settings for screens that don't have the `$post` object defined.

I plan to apply refactoring here and in other places later this week so no action needed for now.